### PR TITLE
Make keyword arguments work on v0.6

### PR DESCRIPTION
**Pull request by joehuchette**
(and hopefully still on v0.5 as well). Ref #2

### CODE REVIEW

File: src/macros.jl [hunk 1]
```diff
@@ -60,8 +60,13 @@ macro polyvariable(args...)
   end
 
   # separate out keyword arguments
-  kwargs = filter(ex->isexpr(ex,:kw), extra)
-  extra = filter(ex->!isexpr(ex,:kw), extra)
+  if VERSION < v"0.6-"
```
Comment (blegat) [inline src/macros.jl]: We will need to be more specific about the version cutoff like what is done here: https://github.com/JuliaOpt/JuMP.jl/commit/b8a321e2862179a00087e65ab80414b49e79d714
However I have no idea how to find the number in the version v"0.6.0-dev.$number". @tkelman How can we find it ?

**Comment by blegat** [comment]
Awesome, thanks !

File: src/macros.jl (hunk 1 above)
Comment (blegat) [inline src/macros.jl]: The version cutoff is `v"0.6.0-dev.1934"` for this one, it is used in https://github.com/JuliaOpt/JuMP.jl/commit/b8a321e2862179a00087e65ab80414b49e79d714 :-P

Comment (blegat) [inline src/macros.jl]: By the way, wouldn't it be easier to do `kwsymbol = VERSION < v"0.6.0-dev.1934" ? :kw : :(=) # changed by julia PR #19868` like done in the JuMP commit mentionned above ?

Comment (joehuchette) [inline src/macros.jl]: Not sure, I find this easier for when we want to go in and drop support for 0.5.

**Pull request merged**